warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__scheduled_trips_in_tu_feed.sql

```diff
@@ -36,15 +36,15 @@ joined AS (
     AND quartet.trip_updates_gtfs_dataset_key IS NOT null
 
     LEFT JOIN fct_daily_scheduled_trips AS scheduled_trips
-      ON idx.date = scheduled_trips.service_date
+      ON idx.date = scheduled_trips.activity_date
```

Comment:
I actually think that for this specific thing we might want to keep `service_date` per this: https://gtfs.org/realtime/reference/#message-tripdescriptor -- it looks like they allow the times past midnight in RT too